cinema/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=63)
+    description = serializers.CharField(required=False, max_length=255)
+    duration = serializers.IntegerField(required=True)
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for a more concise and maintainable code when the serializer directly maps to a model. This approach automatically handles the creation of fields based on the model definition.